src/CompanyInfoGazette.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Worksome\CompanyInfo;
+
+use Illuminate\Http\Client\Response;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Http;
+
+class CompanyInfoGazette
+{
+    /**
+     * Lookup a company name on the English Gazette service, return processed company info.
+     *
+     * TODO: Should probably do a more specific query for the name, if possible?
+     *
+     * @param string $name Name of company to lookup.
+     *
+     * @return array|null Array of company info, or null if request to service failed.
+     */
+    public static function lookupName(string $name): ?array
+    {
+        // @phpstan-ignore-next-line
+        $response = Http::withBasicAuth(config('company-info.services.gazette.key'), '')
```

Comment:
If we move to non-static methods, we could inject the key and base URL into the constructor rather than relying on `config`, again allowing for this class to be used without the need for the Laravel container.